src/mmw/js/src/analyze/controllers.js

```diff
@@ -18,6 +18,11 @@ var AnalyzeController = {
         if (!App.map.get('maskLayerApplied')) {
             App.map.set('maskLayerApplied', true);
         }
+
+        // Clear currProject so that
+        // the ModelingController will know to make a
+        // new project when coming from the Analyze views.
+        App.currProject = null;
```

Comment:
Yes this does indeed break the single project mode. I'm not sure why this line is required at all (since the previous behavior was the same when loading an existing project `/project/XX/scenario/YY/`, going to back to Analyze, then coming forward to Modeling which would create a new project), but if we must keep it then it should only run when not in single project mode:

```javascript
if (!settings.get('activityMode')) {
    App.currProject = null;
}
```